app/models/event.rb

```diff
@@ -0,0 +1,8 @@
+class Event < ActiveRecord::Base
+  validates :lat, presence: true
```

Comment:
I think if anything I side with you on this one. It's more explicit. I used to use `validates` if I had multiple validations on the same attribute, but I've moved away from that.